code/game/machinery/ARES/ARES_interface.dm

```diff
@@ -337,6 +341,74 @@
 			last_menu = current_menu
 			current_menu = "core_security"
 
+		// -- Print ASRS Audit Log -- //
+		if("print_req")
+			playsound = FALSE
+			if(!COOLDOWN_FINISHED(src, printer_cooldown))
+				playsound(src, 'sound/machines/buzz-two.ogg', 15, 1)
+				return FALSE
+			if(!datacore.records_asrs.len)
+				to_chat(user, SPAN_WARNING("There are no records to print!"))
+				playsound(src, 'sound/machines/buzz-two.ogg', 15, 1)
+				return FALSE
+			COOLDOWN_START(src, printer_cooldown, 20 SECONDS)
+			playsound(src, 'sound/machines/fax.ogg', 15, 1)
+			sleep(3.4 SECONDS)
+			var/contents = {"
+						<style>
+							#container { width: 500px; min-height: 500px; margin: 25px auto;  \
+									font-family: monospace; padding: 0; font-size: 130% }  \
+							#title { font-size: 250%; letter-spacing: 8px; \
+									font-weight: bolder; margin: 20px auto }   \
+							.header { font-size: 130%; text-align: center; }   \
+							.important { font-variant: small-caps; font-size = 130%;   \
+										font-weight: bolder; }    \
+							.tablelabel { width: 150px; }  \
+							.field { font-style: italic; } \
+							table { table-layout: fixed }  \
+						</style><div id='container'>   \
+						<div class='header'>   \
+							<p id='title' class='important'>A.S.R.S.</p>   \
+							<p class='important'>Automatic Storage Retrieval System</p>    \
+							<p class='field'>Audit Log</p> \
+						</div><hr>
+						<u>Printed By:</u> [last_login]<br>
+						<u>Print Time:</u> [worldtime2text()]<br>
+						<hr>
+						<center>
+						<table border=1 cellspacing=0 cellpadding=3 style='border: 1px solid black;'>
+						<thead>
+							<tr>
+							<th scope="col">Time</th>
+							<th scope="col">User</th>
+							<th scope="col">Source</th>
+							<th scope="col">Order</th>
+							</tr>
+						</thead>
+						<tbody>
+						"}
+
+			for(var/datum/ares_record/requisition_log/req_order as anything in datacore.records_asrs)
+				if(!istype(req_order))
+					continue
```

Comment:
```suggestion
			for(var/datum/ares_record/requisition_log/req_order as anything in datacore.records_asrs)
```
as anything removes the implicit type check, if the only thing inside of records_asrs is of type datum/ares_record/requisition_log/ then you don't need to do type check.